app/views/rushing_game_logs/_form.html.erb

```diff
@@ -0,0 +1,59 @@
+<%= form_for [player, rushing_game_log] do |f| %>
```

Comment:
As log models like RushingGameLog and PassingGameLog are populated by the scraper, are forms essential functionality for the logs models yet? 

If they can be delayed, consider skipping writing the controller form actions and views (edit, update, create, destroy, new, _form) until they're necessary and focus on building out any functionality that's more crucial to getting the app production-ready.